errands-routes.go

```diff
@@ -152,33 +154,32 @@ func (s *ErrandsServer) updateFilteredErrands(c *gin.Context) {
 			return false
 		}
 	})
-	if err == nil {
-		for _, errand := range errands {
-			if updateReq.Delete == true {
-				err = s.deleteErrandByID(errand.ID)
-				if err != nil {
-					break
-				}
-			} else {
-				if updateReq.Status != "" {
-					_, err = s.UpdateErrandByID(errand.ID, func(e *schemas.Errand) error {
-						e.Status = updateReq.Status
-						return nil
-					})
-					if err != nil {
-						break
-					}
-				}
+
+	var err error
+
+	for _, errand := range errands {
+		if updateReq.Delete {
+			s.deleteErrandByID(errand.ID)
```

Comment:
We are no longer checking this error?

Reply:
Because the error was hard coded to `nil`

> We are no longer checking this error?

Because the error was hard coded to `nil` on line 226 of the original file.